package.json

```diff
@@ -316,14 +316,14 @@
     "tsconfig-paths-webpack-plugin": "4.0.0",
     "typedoc": "0.25.12",
     "typedoc-plugin-markdown": "3.17.1",
-    "typescript": "~5.8.2",
-    "typescript-eslint": "^8.29.0",
+    "typescript": "~5.9.2",
+    "typescript-eslint": "^8.40.0",
     "unist-builder": "^4.0.0",
     "use-sync-external-store": "^1.2.0",
     "verdaccio": "6.0.5",
     "vite": "6.2.0",
     "vitest": "3.0.5",
-    "webpack": "5.99.9",
+    "webpack": "5.101.1",
```

Comment:
This will likely conflict with https://github.com/nrwl/nx/pull/32387/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R326

Also, Angular has updated their version to `5.101.3`: https://github.com/angular/angular-cli/blob/main/packages/angular_devkit/build_angular/package.json#L58